core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerProxy.scala

```diff
@@ -375,6 +377,8 @@ class ContainerProxy(factory: (TransactionId,
 
     case Event(Remove, data: PreWarmedData) => destroyContainer(data)
 
+    case Event(RemovePreWarmedContainer, data: PreWarmedData) => destoryPreWarmedContainer(data)
```

Comment:
I think using Remove message and existing destroyContainer() function is simpler?

Reply:
Reason as below,  i wrote a new method `def destoryPreWarmedContainer(newData: PreWarmedData)`, it seems more clear.